### PR TITLE
Fix #277: Add example of date range parameter usage

### LINKED ISSUE #277: Missing example in "Date Range Parameters"
Just noticed our "[Data Range Parameters](https://redash.io/help/user-guide/querying/query-parameters#Date-Range-Parameters)" doc is missing the example piece:

```
You will typically need to separate them in your query (into different
`WHERE` clauses, e.g.).
```

### PR DESCRIPTION
**Pull request by susodapop**

### CODE REVIEW

**Comment by justinclift** [comment]
This looks 95% good. :smile:

Two minor things to adjust:

* The existing text fragment of `You will typically need to separate them in your query (into different WHERE clauses, e.g.).` needs adjusting
* The SQL example should probably be formatted with normal SQL keyword spacing, as most SQL experienced people will read it easier:

```
SELECT column1, column2, column3
FROM table_name
WHERE table_name.date_column >= {{ myDate.start }}
    AND table_name.date_column <={{ myDate.end }}
```

**Comment by susodapop** [comment]
Thanks Justin. I made some adjustments to the SQL formatting and clarified the token behavior. I also added a bit about how ISO8601 strings are what is sent to the database.

**Comment by justinclift** [comment]
Thanks @susodapop .  The SQL isn't quite right, though it's much closer.  The only that needs adjusting in it now, is to have the first clause of the `WHERE` be on the same line as the `WHERE` itself.  eg:

```
SELECT
	column1,
	column2,
	column3
FROM table_name
WHERE table_name.date_column >= {{ myDate.start }}
	AND table_name.date_column <={{ myDate.end }}
```

At least, that's how it's normally done everyone I've seen. :smile:

File: src/pages/kb/user-guide/querying/query-parameters.md [hunk 1]
```diff
@@ -63,10 +63,29 @@ If you are using Query Parameters of type `Date`, you can set the parameter's de
...
+Here's an example of how you'd integrate a date range parameter into a SQL query. Assume you have created a date range param called `myDate`:
+
+```
+SELECT
+	column1,
+	column2,
+	column3
+FROM table_name
+WHERE
+	table_name.date_column >= {{ myDate.start }}
```
Comment (arikfr) [inline src/pages/kb/user-guide/querying/query-parameters.md]:
```suggestion
	table_name.date_column >= '{{ myDate.start }}'
```

Need to put the date value in quotes.

File: src/pages/kb/user-guide/querying/query-parameters.md [hunk 2]
```diff
@@ -63,10 +63,29 @@ If you are using Query Parameters of type `Date`, you can set the parameter's de
...
+	AND table_name.date_column <={{ myDate.end }}
```
Comment (arikfr) [inline src/pages/kb/user-guide/querying/query-parameters.md]:
```suggestion
	AND table_name.date_column <='{{ myDate.end }}'
```

Same here.

Comment (justinclift) [reply src/pages/kb/user-guide/querying/query-parameters.md]: With a space between the `=` and `'{{ myDate.end }}'`, missing in both your and my example for this line. :wink:

**Comment by susodapop** [comment]
Due to age, I'm rebuilding this into a new PR.

**Pull request closed without merging**